src/screens/Customers/CustomerProfile.tsx

```diff
@@ -81,9 +84,9 @@ function CustomerProfile(props: CustomerProps) {
   const freeUnits = customerTariff ? roundToString(customerTariff?.freeUnits) : UNDEFINED_AMOUNT;
 
   let tariffInfo: CardPropsInfo[] = [
-    { number: fixedTariff, label: 'Fixed Tariff', unit: 'Ks' },
-    { number: unitTariff, label: 'Unit Tariff', unit: 'Ks' },
-    { number: freeUnits, label: 'Free Units', unit: 'kWh' },
+    { number: fixedTariff, label: intl(words.fixed_tariff), unit: intl(words.ks) },
+    { number: unitTariff, label: intl(words.per_unit_tariff), unit: intl(words.ks) },
```

Comment:
Figma & pervious string had this as "Unit Tariff"
![image](https://user-images.githubusercontent.com/35501399/116501976-186fa580-a867-11eb-8e01-d4f39d0ec35b.png)